quickstarts/microsoft.netapp/anf-sap-hana/anf-sap-hana-storage/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "SAP HANA Azure NetApp Files storage",
+  "description": "SAP HANA Azure NetApp Files storage",
+  "summary": "SAP HANA Azure NetApp Files storage",

```

Comment:
Would be great to have more description and summary over this project!

Reply:
Sure

Reply:
Sure